docs/modules/getting_started/examples/code/getting_started.sh.j2

```diff
@@ -52,9 +52,14 @@ echo "Applying the rule file"
 kubectl apply -f simple-rule.yaml
 # end::apply-rule-file[]
 
+# The bundle builder will update the bundle almost immediately, but OPA can take up to
+# max_delay_seconds: 20 (see ConfigMap)
+# to poll the bundle
+sleep 21
```

Comment:
https://github.com/stackabletech/opa-operator/blob/c49c5ae89b71990077364962a86d7f887eb0823f/rust/operator-binary/src/controller.rs#L596

Currently the OPA is configured to randomly poll every 10 to 20 seconds. Worst case you can have ~40 seconds.